src/map/script.c

```diff
@@ -5744,6 +5747,15 @@ BUILDIN_FUNC(warpguild)
 		return SCRIPT_CMD_SUCCESS;
 	}
 
+	switch (type) {
+		case 3:
+			mapindex = mapindex_name2id(str);
+			if (!mapindex)
+				return SCRIPT_CMD_FAILURE;
+			m = map_mapindex2mapid(mapindex);
```

Comment:
not missing, `warp`, `warpparty`, `warpguild` is intended to able warp to other map-server (even in current rA, it'll crash your map-server).
So that's why, in the `pc_job_canenter_map`, I made `m < 0` as `true`.